ygot/render_test.go

```diff
@@ -2240,6 +2246,9 @@ func TestConstructJSON(t *testing.T) {
 		gotietf, err := ConstructIETFJSON(tt.in, &RFC7951JSONConfig{
 			AppendModuleName: tt.inAppendMod,
 		})
+		if err == nil {
+			_, err = json.Marshal(gotietf)
```

Comment:
Can we make this and the one below a separate discussion? i.e., today, if we get the `err != nil` case failing, it's not clear whether it was `ConstructIETFJSON` or `json.Marshal` that threw an error. It'd be good to handle these separately.

Reply:
Done, also restructured the test a bit by using `t.Run()`.

Reply:
Thanks, this LGTM.